README.md

```diff
@@ -4,70 +4,90 @@
 
 # Knife CLC
 
-This is the Chef Knife plugin for CenturyLink Cloud. It gives Knife the ability to manage servers and query for additional resources like datacenters, templates and groups.
+This is the Chef Knife plugin for CenturyLink Cloud. It gives Knife the ability to manage servers and query for additional resources like data centers, templates and groups.
+
+## Requirements
```

Comment:
To be honest, I'm not sure that we need this section. The only thing we can specify is something like:
* Ruby 2.0 or higher

Everything else will be installed automatically as a dependency.